openedx/adg/lms/registration_extension/models.py

```diff
@@ -14,12 +14,13 @@ class ExtendedUserProfile(models.Model):
     """
 
     user = models.OneToOneField(
-        User, unique=True, db_index=True, related_name='extended_profile', on_delete=models.CASCADE)
+        User, db_index=True, related_name='extended_profile', on_delete=models.CASCADE)
 
-    birth_date = models.DateField(verbose_name=_('Birth Date'), null=True)
-    saudi_national = models.BooleanField(verbose_name=_('Saudi National'), null=True)
+    birth_date = models.DateField(verbose_name=_('Birth Date'), null=True, blank=True)
+    saudi_national = models.BooleanField(verbose_name=_('Saudi National'), null=True, blank=True)
 
-    company = models.OneToOneField(BusinessLine, on_delete=models.CASCADE, null=True)
+    company = models.ForeignKey(BusinessLine, verbose_name=_('Company'),
+                                on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
Same, BusinessLine is also required and `blank=True` is not required here

Reply:
Same reason as above